src/adaptor/dlt-adaptor-stdin.c

```diff
@@ -125,10 +126,50 @@ int main(int argc, char* argv[])
             printf("  -a apid      - Set application id to apid (default: SINA)\n");
             printf("  -c ctid      - Set context id to ctid (default: SINC)\n");
             printf("  -t timeout   - Set timeout when sending messages at exit, in ms (Default: 10000 = 10sec)\n");
+            printf("  -v verbosity level - Set verbosity level (Default: INFO, values: FATAL ERROR WARN INFO DEBUG VERBOSE)\n");
             printf("  -h           - This help\n");
             return 0;
             break;
         }
+        case 'v':
+        {
+	  if(!strcmp(optarg, "FATAL"))
+          {
+              verbosity = DLT_LOG_FATAL;
+              break;
+          }
+	  else if(!strcmp(optarg, "ERROR"))
+          {
+              verbosity = DLT_LOG_ERROR;
+              break;
+          }
+	  else if(!strcmp(optarg, "WARN"))
+          {
+              verbosity = DLT_LOG_WARN;
+              break;
+          }
+	  else if(!strcmp(optarg, "INFO"))
+          {
+              verbosity = DLT_LOG_INFO;
+              break;
+          }
+	  else if(!strcmp(optarg, "DEBUG"))
+          {
+              verbosity = DLT_LOG_DEBUG;
+              break;
+          }
+	  else if(!strcmp(optarg, "VERBOSE"))
+          {
+              verbosity = DLT_LOG_VERBOSE;
+              break;
+          } else
+          {
+              printf("Wrong verbosity level, setting default to INFO. Accepted values are: FATAL ERROR WARN INFO DEBUG VERBOSE\n");
```

Comment:
The default is already INFO.  It is not being changed, so instead of "setting default to INFO" this should probably say: "setting to INFO"  or "setting to default value (INFO)" or something similar.